### ISSUE TITLE
Command InstallPlugin raised error

### ISSUE THREAD

**Issue by mherrmann** [fman-community/fman#432]
```
Command 'InstallPlugin' raised error.

Traceback (most recent call last):
File "urllib/request.py", line 1254, in do_open
File "http/client.py", line 1107, in request
File "http/client.py", line 1152, in _send_request
File "http/client.py", line 1103, in endheaders
File "http/client.py", line 934, in _send_output
File "http/client.py", line 877, in send
File "http/client.py", line 1261, in connect
File "ssl.py", line 385, in wrap_socket
File "ssl.py", line 760, in init
File "ssl.py", line 996, in do_handshake
File "ssl.py", line 641, in do_handshake
ssl.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:719)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "fman/impl/plugins/command_registry.py", line 52, in _execute_command
File "/Applications/fman.app/Contents/Resources/Plugins/Core/core/commands/init.py", line 1458, in call
self._plugin_repos = find_repos(topics=['fman', 'plugin'])
File "/Applications/fman.app/Contents/Resources/Plugins/Core/core/github.py", line 11, in find_repos
return list(map(GitHubRepo, _fetch_all_pages(url)))
File "/Applications/fman.app/Contents/Resources/Plugins/Core/core/github.py", line 15, in _fetch_all_pages
data = _get_json(json_url + '&per_page=%d&page=%d' % (page_size, page))
File "/Applications/fman.app/Contents/Resources/Plugins/Core/core/github.py", line 70, in _get_json
return json.loads(_get(url).decode('utf-8'))
File "/Applications/fman.app/Contents/Resources/Plugins/Core/core/github.py", line 73, in _get
return urlopen(url).read()
File "urllib/request.py", line 163, in urlopen
File "urllib/request.py", line 466, in open
File "urllib/request.py", line 484, in _open
File "urllib/request.py", line 444, in _call_chain
File "urllib/request.py", line 1297, in https_open
File "urllib/request.py", line 1256, in do_open
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:719)>
```

**Comment by mherrmann** [comment]
@evanre does it work for you with [fman 1.1.0](http://download.fman.io/1.1.0/fman.dmg)?

**Comment by evanre** [comment]
Hi, @mherrmann thanks for a quick response. Yes, 1.1.0 is work properly.
![image](https://user-images.githubusercontent.com/12615906/41855255-d5676394-789a-11e8-8a4f-03ce217b36f6.png)

**Comment by mherrmann** [comment]
Okay, thanks. That means this bug was introduced by the fix for #236.

**Comment by mherrmann** [comment]
While I cannot reproduce this issue, I believe I was just able to fix it. Will go live with 1.1.2.

**Comment by raguay** [comment]
I just had this happen on my test system that I just installed macOS 10.14 Beta (18A314k). But, it is only installing 1.1.1 right now. I assume 1.1.2 isn't out yet?

**Comment by mherrmann** [comment]
Should be fixed in [fman 1.1.2](https://fman.io/changelog#v1.1.2).
